### PR TITLE
New Navbar done without bootstrap and edited ViewMembers according to UI

### PR DESCRIPTION
**Pull request by DinolanNaidoo**

### CODE REVIEW

File: skietbaan_fe/src/components/NavbarMenu.js [hunk 1]
```diff
@@ -1,136 +1,169 @@
...
+				className="notifications-icon-grey" alt='Notification tab not Selected'></img>
 		}
 	}
 
+	isMore() {
+		return <img src={require('../components/navbar-icons/more-grey.png')}
+			className="more-icon-grey" alt='More icon to expand tray'></img>
+	}
+
+	GoToLeaderboard (){
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/NavbarMenu.js]: Make a generic function for this. 
add a todo. 
Dm me I will help you set it up

File: skietbaan_fe/src/components/NavbarMenu.scss [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    color: #f0f0f0;
+  }
+
+  .App div .navbar .fa{
+    display: grid;
+    text-align: center;
+   }
+
+  .App div .navbar .fa:hover{
+    align-content: center
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/NavbarMenu.scss]: GC

File: skietbaan_fe/src/components/NavbarMenu.scss [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  }
+
+  .more-icon-grey{
+    height: 38px;
+    width: 28px;
+  }
+  
+  .second-row-navbar{
+    display: none;
+
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/NavbarMenu.scss]: cleanup empty lines 78 and 80 also empty line EOF

File: skietbaan_fe/src/components/NavbarMenuUser.js [hunk 4]
```diff
@@ -8,112 +8,139 @@ class NavbarMenuUser extends Component {
...
+				className="notifications-icon-grey" alt='Notification tab not Selected'></img>
 		}
 	}
 
+	isMore() {
+		return <img src={require('../components/navbar-icons/more-grey.png')}
+			className="more-icon-grey" alt='More icon to expand tray'></img>
+	}
+
+	GoToLeaderboard (){
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/NavbarMenuUser.js]: Refer to comment above

File: skietbaan_fe/src/components/ViewMembers.js [hunk 5]
```diff
@@ -106,15 +107,18 @@ class ViewMembers extends Component {
...
+                                </div>}>
+                                    <div className="membership-details">Membership No: <b>{post.memberID}</b>
+                                        <div>Start of Membership: <b>{post.memberStartDate.substring(0, 10)}</b></div></div>
                                 </Collapsible>
                             </td>
                             <td className="second-column">
-                                {this.state.timeLeftOnMembership[index]} Months<br />
-                                <label className={(this.Status(this.state.timeLeftOnMembership[index])) ? "bad":"okay"}>
-                                {post.memberExpiryDate.substring(0, 10)}</label>
+                                <div className="expiry-time-column"><div className={(this.Status(this.state.timeLeftOnMembership[index])) ? "bad" : "okay"}>
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ViewMembers.js]: Spit over multiple lines

Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/ViewMembers.js]: did that

File: skietbaan_fe/src/components/NavbarMenu.scss (hunk 3 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/NavbarMenu.scss]: did that

File: skietbaan_fe/src/components/NavbarMenu.scss (hunk 2 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/NavbarMenu.scss]: deleted it

File: skietbaan_fe/src/components/NavbarMenu.js (hunk 1 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/NavbarMenu.js]: will do

File: skietbaan_fe/src/components/NavbarMenuUser.js [hunk 6]
```diff
@@ -90,51 +86,34 @@ class NavbarMenuUser extends Component {
...
-
 	render() {
 		// rendering the basic navbar within the render class
 		return (
 			<div >
 				<table className="navbar-admin">
 				<tbody>
 					<tr className="first-row-navbar">
-						<td className="columns" onClick={() => this.GoToLeaderboard()}>
+						<td className="columns" onClick={() => this.GoToLeaderboard("/home")}>
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/NavbarMenuUser.js]: You did not update the function call here

Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/NavbarMenuUser.js]: done

**Pull request merged**